From aa75f5136066d239d48a21373b3d96ee12378e8d Mon Sep 17 00:00:00 2001
From: Dengke Du <dengke.du@windriver.com>
Date: Wed, 8 May 2019 17:24:17 +0800
Subject: [PATCH] Install missing conf file

openvzutilstest.conf file is needed by openvzutilstest test.

Upstream-Status: Inapproriate

Signed-off-by: Catalin Enache <catalin.enache@windriver.com>
[KK: Update context for 1.3.5.]
Signed-off-by: Kai Kang <kai.kang@windriver.com>
[MA: Update context for v4.3.0]
Signed-off-by: Mark Asselstine <mark.asselstine@windriver.com>
[DDU: Update context for v5.3.0]
Signed-off-by: Dengke Du <dengke.du@windriver.com>

---
 tests/Makefile.am | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/Makefile.am b/tests/Makefile.am
index 4a808dd..0c3e799 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -173,6 +173,7 @@ EXTRA_DIST = \
 	$(NULL)
 
 test_helpers = commandhelper ssh
+test_misc =
 test_programs = virshtest sockettest \
 	virhostcputest virbuftest \
 	commandtest seclabeltest \
@@ -311,6 +312,7 @@ endif WITH_LXC
 
 if WITH_OPENVZ
 test_programs += openvzutilstest
+test_misc += openvzutilstest.conf
 endif WITH_OPENVZ
 
 if WITH_ESX
@@ -1551,7 +1553,7 @@ endif  ! WITH_LINUX
 
 buildtest-TESTS: $(TESTS) $(test_libraries) $(test_helpers)
 
-PTESTS = $(TESTS) $(test_helpers) test-lib.sh virschematest
+PTESTS = $(TESTS) $(test_helpers) $(test_misc) test-lib.sh virschematest
 
 install-ptest:
 	list='$(TESTS) $(test_helpers) test-lib.sh virschematest'
